src/Datasource/InvalidPropertyInterface.php

```diff
@@ -25,18 +25,4 @@
  */
 interface InvalidPropertyInterface
 {
-    /**
-     * Sets a field as invalid and not patchable into the entity.
-     *
-     * This is useful for batch operations when one needs to get the original value for an error message after patching.
-     * This value could not be patched into the entity and is simply copied into the _invalid property for debugging purposes
-     * or to be able to log it away.
-     *
-     * @param string|array|null $field The field to get invalid value for, or the value to set.
-     * @param mixed|null $value The invalid value to be set for $field.
-     * @param bool $overwrite Whether or not to overwrite pre-existing values for $field.
-     * @return $this|mixed
-     * @deprecated 3.5.0 Use getInvalid()/getInvalidField() and setInvalid()/setInvalidField() instead.
```

Comment:
Are we going to add the new methods in separate pull requests?

Reply:
Yes.